src/test/java/SeleniumTestngTest.java

```diff
@@ -1,24 +1,192 @@
+
+        logger.info("Clicking 'Logout' button");
+        WebElement logoutBtn = browser.findElement(By.xpath("//i[contains(text(), 'Logout')]"));
+        logoutBtn.click();
+
+        WebElement loginHeader = browser.findElement(By.xpath("//h2"));
+        Assert.assertEquals(loginHeader.getText(), "Login Page");
+        Assert.assertEquals(browser.getCurrentUrl(), "https://the-internet.herokuapp.com/login");
     }
 
-    // Write the rest of TEST METHODS according to the task here, each method checking one scenario described in README.md file
-    // In the end you should have a set of test methods each of them describing some specific scenario
+    @Test
+    public void dragAndDropTest() throws InterruptedException {
+//    На сторінці "Drag and Drop" перетягніть eлемент A на місце елемента B,
+//    та переконайтеся що вони помінялися місцями
+        logger.info("Redirecting to the 'Drag and Drop' page");
+        WebElement dragAndDropLink = browser.findElement(By.linkText("Drag and Drop"));
+        dragAndDropLink.click();
+        WebElement elementA = browser.findElement(By.id("column-a"));
+        WebElement elementB = browser.findElement(By.id("column-b"));
+
+        logger.info("Moving element A to the position of element B");
+        Actions actions = new Actions(browser);
+        actions.dragAndDrop(elementA, elementB).build().perform();
+
+        Thread.sleep(1000);
+
+        WebElement firstElementHeader = browser.findElement(By.xpath("//*[@id=\"column-a\"]/header"));
+        WebElement secondElementHeader = browser.findElement(By.xpath("//*[@id=\"column-b\"]/header"));
+
+        Assert.assertEquals(firstElementHeader.getText(), "B");
+        Assert.assertEquals(secondElementHeader.getText(), "A");
+    }
 
+    @Test
+    public void verifyHorizontalSliderTest() throws InterruptedException {
+//    На сторінці "Horizontal Slider" перетягніть слайдер в положення, що відрізняється від 0 і перевірте,
+//    що необхідне значення дійсно було встановлене на слайдері
+        logger.info("Redirecting to the 'Horizontal Slider' page");
+        WebElement horizontalSliderLink = browser.findElement(By.linkText("Horizontal Slider"));
+        horizontalSliderLink.click();
+
+        WebElement slider = browser.findElement(By.xpath("//input"));
+        int shift = new Random().nextInt(50);
+
+        logger.info("Moving slider to {} pixel(s) right", shift);
+        Actions actions = new Actions(browser);
+        actions.clickAndHold(slider)
+                .moveByOffset(shift, 0)
+                .release()
+                .perform();
+
+        Thread.sleep(100);
+        WebElement sliderRangeElement = browser.findElement(By.id("range"));
+        double sliderValue = Double.parseDouble(sliderRangeElement.getText());
+
+        logger.info("Slider value after moving: {}", sliderValue);
+        Assert.assertTrue(sliderValue > 0.0d, "Slider value should be greater than 0");
```

Comment:
This assertion is good, but not enough, as probably you'd like to make sure the slider was indeed moved to the exact value, not just any non-zero value). This is possible - if you look what exactly changes in DOM (keep DevTools opened and move the slider manually to see how the final value is represented in DOM

Reply:
One more hint: you're moving slider by some amount of pixels which doesn't guarantee you the precise value - so maybe there's another way of doing it so the precision will be kept